### ISSUE TITLE
Duplication Glitch using Different mods

### ISSUE THREAD

**Issue by QuImUfu** [DrParadox7/Lost-Era-Modpack#93]
### Your Pack Version

1.6.3

### Type of Instance

Multiplayer

### Your Expectation

I build a funny/quite overly complex contraption to feed endoflames looking like this:
```
      R
      DA
       C
TBBBBBBB*
B       B

R: Remote Comparator watching the Item Dropper below causing inserted single items to be ejected immediatly.
D: Item Dropper
A: Item Allocator moving a single Item a time to the Item Dropper
C: Item Collector
T: Translocation Plate, beaming the item to *
B: Conveyor Belt from ExtraUtilities going to the left
```

Its intended purpose was to feed endoflames with items we put into the Item Allocator, which it does. 

### The Reality

But if no one is online, that contraption starts duplicating my fuel. i.e. when I rejoin the Item Allocator is full and ~an inventory worth of fuel is stuck right before it.

Chunk border is between `D` and `A` and it is plausible that only the left chunk is loaded at the necessary level to process entities by the world spawn.  

I can upload my world or send you to the server address if you want me to.

### Your Proposal

IDK, if you care, figure out how it works and fix it, or, if you don't care, close this issue.

### Final Checklist

- [X] I have searched this issue tracker and there is nothing similar already.
- [ ] I can reproduce this problem consistently by follow the exact steps I described above, or this does not need reproducing, e.g. recipe loophole. It seems to happen quite consistently, but maybe not always.

**Comment by DrParadox7** [comment]
I'd prefer a screenshot of the setup to investigate this

**Comment by QuImUfu** [comment]
![dupe setup](https://github.com/DrParadox7/Lost-Era-Modpack/assets/9885301/4174b615-d95d-4d8c-8853-e098742a9664)

**Comment by DrParadox7** [comment]
Should be fixed in 1.6.5
